exercises/raindrops/uRaindropsTest.pas

```diff
@@ -0,0 +1,179 @@
+unit uRaindropsTest;
+
+interface
+uses
+  DUnitX.TestFramework;
+
+type
+
+  [TestFixture]
+  RaindropsTest = class(TObject)
+  public
+    [Test]
+    procedure the_sound_for_1_is_1;
+
+    [Test]
+    [Ignore('Comment these lines to run these tests')]
```

Comment:
I'll take some time later and insert commented _`// [Ignore('Comment the "[Ignore]" statements to run these tests')]`_  I will do away with the comment that now appears on all the second tests.  Sound OK?